app/lib/channels/remove.rb

```diff
@@ -0,0 +1,23 @@
+module Hammeroids
+  module Channels
+    # Broadcasts player removal
+    class Remove
+      def initialize(channel, id)
+        @channel = channel
+        @id = id
+      end
+
+      def broadcast
+        @channel.push(to_json)
+      end
+
+      def attributes
+        { type: "remove", payload: {id: @id}}
```

Comment:
hashes should have spaces around keys.

` { type: "remove", payload: { id: @id } }`